Makefile

```diff
@@ -0,0 +1,64 @@
+ # Go parameters
+GOCMD=go
+GOBUILD=$(GOCMD) build
+GOCLEAN=$(GOCMD) clean
+GOTEST=$(GOCMD) test
+GOGET=$(GOCMD) get
+BINARY_NAME=frontman
+# BINARY_UNIX=$(BINARY_NAME)_unix
+
+all: test build
+build: 
+	$(GOBUILD) -o $(BINARY_NAME) -v ./cmd/frontman/...
+
+test: 
+	$(GOTEST) -v ./...
+
+clean: 
+	$(GOCLEAN)
+	rm -f $(BINARY_NAME)
+	# rm -f $(BINARY_UNIX)
+
+run:
+	$(GOBUILD) -o $(BINARY_NAME) -v ./cmd/frontman/...
+	./$(BINARY_NAME)
+
+vendor:
+	go mod vendor
```

Comment:
AFAIK we don't use modules yet. What is the purpose of this command?

Reply:
my bad